### PR TITLE
Restyle

### PR DESCRIPTION
**Pull request by Explosion-Scratch**

### CODE REVIEW

**Comment by Explosion-Scratch** [comment]
Don't merge yet, WIP

File: components/Nav.vue [hunk 1]
```diff
@@ -13,12 +13,12 @@
           v-if="$auth.loggedIn"
           class="flex justify-center content-center flex-col"
         >
-          <div class="flex flex-row">
-            <span class="text-base">Hi, {{ $auth.user.name }}</span>
+          <div class="flex flex-row text-gray-600">
+            <span class="text-base p-1">Hi, <span class="font-semibold text-red-700">{{ $auth.user.name }}</span></span>
```
Comment (GrahamSH-LLK) [inline components/Nav.vue:17]: You seem to have an extra span

File: components/Nav.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="p-2 flex justify-between ">
+  <div class="p-2 py-3 flex justify-between shadow-lg bg-white">
```
Comment (GrahamSH-LLK) [inline components/Nav.vue:2]: Now that it's white, it needs a dark: value

File: pages/settings.vue [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
-      </section>
+ <div class="flex justify-center items-center h-screen w-screen flex-col bg-gradient-to-tr from-red-400 via-blue-300 to-purple-400">
+    <div class="settings w-5/6 max-w-lg bg-white p-5 rounded bg-opacity-60">
+      <h1 class="font-extralight text-3xl text-center mb-1 text-gray-800">Settings</h1>
+      <hr class="block h-0.5 bg-gradient-to-r from-transparent via-gray-400 to-transparent w-full">
+      <div class="section danger p-4 rounded-lg my-4 w-full border-red-400 border-2 flex flex-col">
+        <h2 class="font-semibold text-xl text-red-700">Danger zone</h2>
+        <i class="font-light text-gray-800 mb-2">These buttons just email support, feel free to click.</i>
+        <div class="buttons flex w-full">
+          <a href="mailto:graham@ejenda.org?subject=I%20would%20like%20to%20delete%20my%20Ejenda%20account&body=My%20username%20is%20%5Byour_username%5D" class="mx-1 flex-1 p-2 border-red-600 hover:text-red-100 hover:bg-red-900 active:scale-95 border-2 rounded text-white bg-red-600 transition-all duration-300">
```
Comment (GrahamSH-LLK) [inline pages/settings.vue:10]: You can dynamically edit the username. Also, add the secret db id to the email.

**Review by GrahamSH-LLK — CHANGES_REQUESTED** [review]
Looks great! Here are a few nits

File: components/Nav.vue (hunk 1 above)
Comment (Explosion-Scratch) [reply components/Nav.vue:17]: That's to color the username differently

File: pages/settings.vue (hunk 3 above)
Comment (Explosion-Scratch) [reply pages/settings.vue:10]: How do I do that?

**Comment by Explosion-Scratch** [comment]
## Images:

#### Dark mode:
image

#### Dark logged in:
image

#### Light mode
image

#### Light logged in
image

File: components/Nav.vue (hunk 1 above)
Comment (GrahamSH-LLK) [reply components/Nav.vue:17]: Ah, I'm blind

**Comment by GrahamSH-LLK** [comment]
image

**Comment by Thingyman-DEV** [comment]
@Explosion-Scratch , are you still working on this, or do you want me and Graham to review?

**Comment by Explosion-Scratch** [comment]
> @Explosion-Scratch , are you still working on this, or do you want me and Graham to review?

Sorry, forgot about it, wait 3 days or so, I'll try to work on it

**Comment by Thingyman-DEV** [comment]
> > @Explosion-Scratch , are you still working on this, or do you want me and Graham to review?
> 
> Sorry, forgot about it, wait 3 days or so, I'll try to work on it

No problem, all good.

**Comment by Thingyman-DEV** [comment]
Closing as most changes have now been implemented as of 2.1, settings page will be redesigned soon

**Pull request closed without merging**